src/plugin/plugin.ts

```diff
@@ -427,14 +427,17 @@ export interface PurchasesStoreProduct {
   readonly title: string;
   /**
    * Price of the product in the local currency.
+   * Contains the currency code value of defaultOption for Google Play.
```

Comment:
Total nitpick, but I kinda would prefer:
```suggestion
   * For Google Play, contains the currency code value of the defaultOption.
```
Just so it's clear from the start of the line that only applies to google play. Same for the others.